### PR TITLE
Bazel 6 JVM

### PR DESCRIPTION
**Pull request by brocollie08**

### CODE REVIEW

File: conf.bzl [hunk 1]
```diff
@@ -1,11 +1,8 @@
 load("//bazel:conf.bzl", _skylib = "skylib")
 load("//apple:conf.bzl", _apple = "apple")
-load("//javascript:conf.bzl", _javascript = "javascript")
-load("//kotlin:conf.bzl", _kotlin = "kotlin")
 load("//distribution:conf.bzl", _distribution = "distribution")
 
 skylib = _skylib
 apple = _apple
 distribution = _distribution
 javascript = _javascript
```
Comment (sugarmanz) [inline conf.bzl:8]: This whole file should be deleted, yeah?

File: MODULE.bazel [hunk 2]
```diff
@@ -17,6 +18,34 @@ rules_ts_ext.deps()
...
+        "io.github.gradle-nexus:publish-plugin:1.1.0",
+        "ch.qos.logback:logback-classic:1.2.11",
+        "com.eclipsesource.minimal-json:minimal-json:0.9.5",
+        "com.electronwill.night-config:core:3.6.5",
+        "com.electronwill.night-config:toml:3.6.5",
+        "com.google.http-client:google-http-client:1.34.2",
+        "info.picocli:picocli:4.3.2",
+        "org.apache.commons:commons-compress:1.21",
+        "org.jsoup:jsoup:1.16.1",
+        "org.zeroturnaround:zt-exec:1.10",
```
Comment (sugarmanz) [inline MODULE.bazel:39]: do we need all of these? i only see these used:
```
    "@rules_player_maven//:com_github_ajalt_clikt_clikt_jvm",
    "@rules_player_maven//:io_github_gradle_nexus_publish_plugin",
    "@rules_player_maven//:ch_qos_logback_logback_classic",
```

File: kotlin/private/junit_test.bzl [hunk 3]
```diff
@@ -0,0 +1,144 @@
...
+             for artifact_id in JUNIT_PLATFORM_ARTIFACT_ID_LIST
+         ],
+         **kwargs
+     )
+
+def _format_maven_jar_name(group_id, artifact_id):
+    return ("%s_%s" % (group_id, artifact_id)).replace(".", "_").replace("-", "_")
+
+def _format_maven_jar_dep_name(group_id, artifact_id):
+    return "@rules_player_maven//:%s" % _format_maven_jar_name(group_id, artifact_id)
```
Comment (sugarmanz) [inline kotlin/private/junit_test.bzl:144]: This fails now, right? I don't see the junit dependencies in the maven install

File: kotlin/example/.bazelversion
Comment (sugarmanz) [inline kotlin/example/.bazelversion:1]: This should be copied -- not moved

**Review by sugarmanz — APPROVED** [review]

**Comment by intuit-svc** [comment]
:rocket: PR was released in [`v1.0.0`](https://github.com/player-ui/rules_player/releases/tag/v1.0.0) :rocket:

**Pull request merged**